usr.sbin/daemon/daemon.c

```diff
@@ -534,14 +553,14 @@ daemon_sleep(time_t secs, long nsecs)
 
 static void
 open_pid_files(const char *pidfile, const char *ppidfile,
```

Comment:
why are these two not part of the state?

Reply:
they should be, I was trying to be minimalist with my change and moved only those variables that are needed to git rid of `goto exit`

Reply:
another answer to this question is this - these are only necessary to until those files are opened and we have their descriptors, so storing names and descriptors feels a bit redundant..